R/utility.R

```diff
@@ -333,7 +333,8 @@ getJobResult <- function(jobId = "", ...) {
 
   if (!is.null(args$container)) {
```

Comment:
We can remove this case and the elipsis and ...

Since the job id is the container name